.travis.yml

```diff
@@ -28,6 +28,8 @@ before_install:
 install: echo "no install"
 
 stages:
+  - name: ADF linking
+    if: type = pull_request AND commit_message =~ \[link-adf:*
```

Comment:
I'm not sure the second expression is needed here: `commit_message =~ \[link-adf:*`, since this condition is checked either in the script.
Also not sure, how the clone ADF are shared between the jobs, without using travis worksapces. Originally I would prefer to do it in before_install, just as in the monorepo. I don't think we need a specific job for this.

Reply:
As you suggested I moved linking part to `before_install` script as in monorepo